javav2/usecases/creating_lambda_ppe/Readme.md

```diff
@@ -0,0 +1,892 @@
+
+        this.key = key;
+     }
+
+    public String getKey() {
+        return this.key;
+    }
+
+    public void setDate(String date) {
+
+        this.date = date;
+    }
+
+    public String getDate() {
+        return this.date;
+    }
+
+    public String getItem() {
+        return this.item;
+    }
+
+    public void setItem(String item) {
+
+        this.item = item;
+    }
+
+    public String getItemDescription() {
+        return this.itemDescription;
+    }
+
+    public void setItemDescription(String itemDescription) {
+
+        this.itemDescription = itemDescription;
+    }
+
+    public String getCoverDescription() {
+        return this.coverDescription;
+    }
+
+    public void setCoverDescription(String coverDescription) {
+
+        this.coverDescription = coverDescription;
+    }
+
+    public String getConfidence() {
+        return this.confidence;
+    }
+
+    public void setConfidence(String confidence) {
+
+        this.confidence = confidence;
+     }
+    }
+
+### GearItem class
+
+The **GearIten** class represents the model in this use case. Its stores data retrieved from the Amazon Rekognition service. 
+
+     package com.example.ppe;
+
+     public class GearItem {
+
+     private String key;
+     private String name;
+     private String itemDescription;
+     private String bodyCoverDescription;
+     private String confidence ;
+
+     public void setItemDescription (String itemDescription) {
+
+        this.itemDescription = itemDescription;
+     }
+
+     public String getItemDescription() {
+
+        return this.itemDescription;
+     }
+
+     public void setBodyCoverDescription (String bodyCoverDescription) {
+        this.bodyCoverDescription = bodyCoverDescription;
+     }
+
+     public String getBodyCoverDescription() {
+
+        return this.bodyCoverDescription;
+    }
+
+    public void setKey (String key) {
+        this.key = key;
+    }
+
+    public String getKey() {
+        return this.key;
+    }
+
+    public void setName (String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setConfidence (String confidence) {
+        this.confidence = confidence;
+    }
+
+    public String getConfidence() {
+        return this.confidence;
+    }
+   }
+
+
+### PPEHandler class
+
+This Java code represents the **PPEHandler** class. This class reads a value that specifies which Amazon S3 bucket to read the images from. The **s3Service.ListBucketObjects** method returns a **List** object where each element is a string value that represents the object key. For each image in the bucket, a byte array is obtained by calling the **s3Service.getObjectBytes** method. Then an **ArrayList** is obtained by calling the **photos.detectLabels** method. Finally the **ArrayList** object is added to another collection. The data that specifies PPE gear is then persisted in an DynamoDB table and emailed to a user. 
```

Comment:
a byte array is obtained by calling the **s3Service.getObjectBytes** method -> the **s3Service.getObjectBytes** method obtains a byte array 
(avoid passive construction)

Reply:
fixed